app/webhook/tests.py

```diff
@@ -16,36 +21,49 @@ def test_webhook_200(self, mock_requests):
             mock_response.status_code = 200
             mock_response.content = b"ok"
             mock_requests.post.return_value = mock_response
-            
+
             invoice_start_date = "01-01-2024"
             threshold = 75
 
-            response = self.client.post("/webhook/scaleway/secret_token_xyz", {"invoice_start_date": invoice_start_date, "threshold": threshold})
+            response = self.client.post(
+                "/webhook/scaleway/secret_token_xyz",
+                data={"invoice_start_date": invoice_start_date, "threshold": threshold},
+                content_type="application/json",
```

Comment:
Par défaut, `client.post` met un content type à `application/x-www-form-urlencoded`, ce qui explique que le test fonctionnait et que je n'avais pas vu le problème.

Maintenant le content_type du test est aligné avec celui utilisé par [scaleway](https://www.scaleway.com/en/docs/console/account/how-to/use-billing-alerts/#webhook-api-call).